servicebox-app/src/main/resources/templates/claims/table.html

```diff
@@ -12,14 +12,8 @@
     </thead>
     <tbody>
     {#for claim in claims}
-    {#include claims/item.html claim=claim /}
+      {#include claims/item.html claim=claim /}
     {/for}
     </tbody>
   </table>
-</div>
-<footer class="footer">
-  <id class="container">
-    <!-- Showing the number of claims. Returned by the template key: items of ClaimResource -->
-    <strong>{items}</strong> {#if items == 1}item{#else}items{/if}
-  </id>
-</footer>
```

Comment:
I moved the footer here because when doing a search by a service requested or a claim name, the number of items was not updated according to the number of claims displayed in the table.